base/complex.jl

```diff
@@ -263,6 +263,20 @@ julia> conj(1 + 3im)
 conj(z::Complex) = Complex(real(z),-imag(z))
 abs(z::Complex)  = hypot(real(z), imag(z))
 abs2(z::Complex) = real(z)*real(z) + imag(z)*imag(z)
+
+"""
+    mod(z::Complex,n::Real)
+Returns a complex number whose real part and imaginary parts are old real part modulo n and old imaginary part modulo n respectively.
+# Examples
+```jldoctest
+julia> mod(12+10im,5)
+2 + 0im
+julia> mod(3+11im,7)
+3 + 4im
+```
+"""
+mod(z::Complex,n::Real)=Complex(mod(real(z),n),mod(imag(z),n))
```

Comment:
It's not clear that we need a new docstring, since the old docstring for `mod(x, y)` seems to cover this case.

However, for consistency we should have similar methods for `div` and `rem`